dmHelper/dmHelper.py

```diff
@@ -122,7 +122,7 @@ async def on_member_join(self, member: discord.Member):
     async def _message_listener(self, message: discord.Message):
         if not isinstance(message.channel, discord.DMChannel):
             return
```

Comment:
This should be tabbed in. Did we miss this before?

Reply:
I'm not seeing what you're talking about.

Reply:
It might be showing weird on mobile. Is the return indented?

Reply:
Yeah I think it's just displaying weird

Reply:
Yes